api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -73,6 +73,10 @@ type VirtualMachineImageStatus struct {
 	// Deprecated
 	PowerState    string `json:"powerState,omitempty"`
 
+	// ImageCompatibility indicates whether the VirtualMachineImage's OVF is prepared by VI admin or not and is supported
```

Comment:
Do we only plan to set this for the OVF type images? Maybe we should reword this so this does not reference the image format?

Reply:
This field is removed and the documentation is also updated